### PR TITLE
Removing andes specific checks

### PR DESCRIPTION
**Pull request by ruwany**

### CODE REVIEW

File: components/event/org.wso2.carbon.event.core/src/main/java/org/wso2/carbon/event/core/internal/builder/EventBrokerBuilderDS.java [hunk 1]
```diff
@@ -57,7 +58,11 @@ protected void activate(ComponentContext context) {
         boolean isQpidBundlePresent = false;
         final BundleContext bundleContext = context.getBundleContext();
         for (Bundle bundle : bundleContext.getBundles()) {
-            if (bundle.getSymbolicName().equals("org.wso2.carbon.andes") || bundle.getSymbolicName().equals("org.wso2.carbon.qpid") ) {
+//            if (bundle.getSymbolicName().equals("org.wso2.carbon.andes") || bundle.getSymbolicName().equals("org.wso2.carbon.qpid") ) {
+//                isQpidBundlePresent = true;
+//                break;
+//            }
+            if (bundle.getSymbolicName().equals("org.wso2.carbon.qpid") ) {
```
Comment (Nipuni) [inline components/event/org.wso2.carbon.event.core/src/main/java/org/wso2/carbon/event/core/internal/builder/EventBrokerBuilderDS.java]: check formatting in the if check

**Comment by Nipuni** [comment]
Please get this reviewed and also add a carbon jira for this

**Comment by ruwany** [comment]
Fixed comments.

**Comment by ramith** [comment]
+1 for this change. (we don't use this component anymore anyway in MB therefore no risk)

**Comment by Nipuni** [comment]
Please add a carbon-jira for the fix

**Comment by Nipuni** [comment]
COMMONS-5

**Pull request merged**